src/classes/CRLP_RollupAccount_TEST.cls

```diff
@@ -67,7 +67,9 @@ private class CRLP_RollupAccount_TEST {
         String filterGroupId1 = CMT_UnitTestData_TEST.getNewRecordId();
         String filterGroupId2 = CMT_UnitTestData_TEST.getNewRecordId();
 
-        String dtValue = DateTime.now().addMonths(-100).addDays(-1).format('yyyy-MM-dd');
+        DateTime dt = DateTime.now().addMonths(-100).addDays(-1);
+        String dtValue = dt.format('yyyy-MM-dd', UserInfo.getTimezone().toString());
+
```

Comment:
This looks perfect! I just tested with a ton of debug/anon stuff and I believe this will successfully avoid future failures based on the date being off by one because of time zone. The only thing I'd suggest is adding a comment to remind future devs (future us) why we have to go through the gymnastics and can't use System.today() that already accounts for time zone. 
Something like... 
// Must use DateTime to get .format(), so we have to explicitly ensure it's in the same time zone as System.today() dates used elsewhere that are already in user's time zone